src/NUnitConsole/nunit3-console.tests/App.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8" ?> 
+<configuration>
+  <appSettings>
+    <add key="test.setting" value="54321"/>
+  </appSettings>
+</configuration>
```

Comment:
This file is included within the "add agent to msi" commit - is it included accidentally? I'm not sure what the purpose of this is.

Reply:
I had to add it to make all tests pass. There's a test that reads it. In fact, this should eventually be cleaned up and we should stop using `.config` files entirely for .NET Core as a part of reworking RuntimeFramework. `.runtimeconfig.json` is where this should end up. 

Hard to figure out the boundaries of a PR... i.e. where do you stop and create a new issue. Personally, I'd prefer not to deal with this one while we still allow `--inprocess` because that makes RuntimeFramework more complicated. I do think that RuntimeFramework is a good place for me to exercise my preference for tiny PRs. :smiley: